ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -294,7 +314,7 @@ protected void doInBackgroundGuarded(Void... params) {
       // an SQLite DB and forwards parameters to it without doing any parsing / validation.
       try {
         Cursor photos = resolver.query(
-            Images.Media.EXTERNAL_CONTENT_URI,
+            MediaStore.Files.getContentUri("external"),
```

Comment:
This is because querying against `Images.Media.EXTERNAL_CONTENT_URI` will only gives you images. So we need to query against the "whole external content uri" and use `MediaStore.Files.FileColumns.MEDIA_TYPE` to filter out what we need, either photos or videos or both.